src/xcode/ENA/ENA/Source/Scenes/Settings/NotificationSettings/NotificationSettingsViewController.swift

```diff
@@ -46,6 +46,11 @@ class NotificationSettingsViewController: UIViewController, UITableViewDataSourc
 		notificationSettings()
 	}
 
+	override func viewWillAppear(_ animated: Bool) {
```

Comment:
Same question here. See above.

Reply:
See answer on DataDonationViewController.
NotificationSettingsViewController does not inherit from UITableViewController, and I saw no adverse effect when I moved these lines.